.lando.yml

```diff
@@ -13,6 +13,7 @@ services:
         SIMPLETEST_DB: 'mysql://drupal9:drupal9@database/drupal9'
         SIMPLETEST_BASE_URL: 'http://appserver'
         MINK_DRIVER_ARGS_WEBDRIVER: '["chrome", {"browserName":"chrome","chromeOptions":{"args":["--disable-gpu","--disable-dev-shm-usage","--headless","--no-sandbox"]}}, "http://chromedriver:9515"]'
+        XDEBUG_MODE: debug
```

Comment:
Does this environment variable have any effect if the xdebug extension is disabled?